.github/workflows/test-basic-failed-only-or8n.yml

```
@@ -0,0 +1,60 @@
+name: demo.playwright.failed-only-or8n
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test-or8n:
+    strategy:
+      fail-fast: false
+      matrix:
+        shard: [1, 2, 3]
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    container: mcr.microsoft.com/playwright:latest
+    env:
+      CURRENTS_PROJECT_ID: bnsqNa
+      CURRENTS_RECORD_KEY: ${{ secrets.CURRENTS_RECORD_KEY }}
+      CURRENTS_CI_BUILD_ID: or8n-${GITHUB_REPOSITORY}-${GITHUB_RUN_ID}-${GITHUB_RUN_ATTEMPT}
```

```suggestion
      CURRENTS_CI_BUILD_ID: or8n-${{ github.repository }}-${{ github.run_id }}-${{ github.run_attempt }}
```

@vCaisim this build Id wasn't working. Looks like you had fixed it in the other files.

---

Thanks, @twk3! Didn't know that default workflow variables aren't available at that level.